tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_sfpu/llk_math_eltwise_unary_sfpu_macros.h

```diff
@@ -0,0 +1,176 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "llk_math_eltwise_unary_sfpu_init.h"
+#include "llk_math_eltwise_unary_sfpu_params.h"
+
+// For ops that require only the init function
+#define SFPU_UNARY_KERNEL_INIT(OP, APPROXIMATE) llk_math_eltwise_unary_sfpu_init<SfpuType::OP, APPROXIMATE>();
```

Comment:
In macros it's typical to use fully qualified names by starting with `::` so as to avoid potentially different meaning depending on the scope in which it's expanded.

Also it's typical to omit the semicolon from macro expansions so that they're syntactically required where pasted.